src/resources/config/seotools.php

```diff
@@ -6,10 +6,11 @@
          * The default configurations to be used by the meta generator.
          */
         'defaults'       => [
-            'title'       => "It's Over 9000!", // set false to total remove
-            'description' => 'For those who helped create the Genki Dama', // set false to total remove
-            'separator'   => ' - ',
-            'keywords'    => [],
+            'title'        => "It's Over 9000!", // set false to total remove
+            'description'  => 'For those who helped create the Genki Dama', // set false to total remove
+            'separator'    => ' - ',
+            'keywords'     => [],
+            'canonical'    => '', // Leave blank for using Url::current(), set false to total remove
```

Comment:
leave the default as 'false', not to impact those who already use the package today. Otherwise it would be a version of break.